app.py

```diff
@@ -1,21 +1,25 @@
 from flask import Flask, send_from_directory, request, jsonify
 from utils.data import get_criteria, get_pets
+from utils.ahp import AnalyticalHierarchyProcess
 import os
+import sys
 
-criteria = None
 pets = None
 result = None
+criteria = None
```

Comment:
I think there's no need for the global variables `pets`, `result` and `criteria`. Maybe let's get rid of them and add method to `AnalyticHierarchyProcess` that checks wether the `criteria_matrix` and `pets_matrix` have been set and use it in `/api/available` route.